samples/subsys/fs/littlefs/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -6,13 +6,30 @@
 
 /delete-node/ &storage_partition;
 
+/ {
+	fstab {
```

Comment:
I think we should put this under /chosen by convention.

Reply:
DT spec 3.5:

> The /chosen node does not represent a real device in the system but describes parameters chosen or specified by the system firmware at run time. It shall be a child of the root node.

It only describes properties as being part of this node.  I'm fine with moving it here if that's compatible with standard use.